designs/disruption-controls.md

```diff
@@ -0,0 +1,118 @@
+# Disruption Controls
```

Comment:
Should this doc cover detail on how we got to refining our disruption controls down to these API fields as well as the use-cases that these controls are going to cover explicitly.

Reply:
Good callout. Added some bit about this here.